coriolis/tests/cmd/test_api.py

```diff
@@ -0,0 +1,81 @@
+# Copyright 2024 Cloudbase Solutions Srl
+# All Rights Reserved.
+
+from unittest import mock
+
+import sys
```

Comment:
NIT: No need to separate `sys` here. It can go in the same group as the `unittest` import